components/admin_console/custom_url_schemes_setting.jsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import * as Utils from 'utils/utils';
+
+import Setting from './setting';
+
+export default class TextSetting extends React.Component {
+    static get propTypes() {
+        return {
+            id: PropTypes.string.isRequired,
+            value: PropTypes.array.isRequired,
+            onChange: PropTypes.func,
+            disabled: PropTypes.bool,
+            setByEnv: PropTypes.bool.isRequired,
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            value: this.arrayToString(props.value),
+        };
+    }
+
+    stringToArray = (str) => {
+        return str.split(',').map((s) => s.trim()).filter(Boolean);
```

Comment:
`.filter(Boolean)` is new to me that I needed to google on how it's working.  It looks neat.

Reply:
It's just the same as passing `.filter((item) => Boolean(item))`. I never realized before that `Boolean` was just a function so we could write it this way